content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In the [model registry chapter](/doc/start/model-management/model-registry) we
+registered the model in the model registry and assigned it to some lifecycle
+stages. In this chapter, we will learn how to access and use models and how to
+use the model registry to trigger automated CICD model workflows.
+
+If you are using the example repository, the models are already versioned on a
+publicly readable DVC remote so you can access the model from there and use it.
+If you are instead using your own repository you need to set up your own DVC
+remote and push the data (including models) there. Have a look at our
+[Data management guide](/doc/start/data-management/data-versioning#configuring-a-remote)
+to see how this is done.
+
+## Downloading models
+
+It is useful to download model artifacts for example for local testing or for
+use in CICD workflows. With models versioned by DVC this can be done easily by
+using the Studio UI.
+
+Go to the detailed view of your model, select the desired model version under
+the "Version info" and then click on the "Access model" button.
+
+Studio will present you with several ways of downloading models - with the CLI,
+in Python code and directly from your web browser. You can see all the web
+browser download steps here:
+
+<video width="99%" height="540" autoplay loop muted>
+    <source src="/img/mr-studio-download-model.webm" type="video/webm">
+</video>
+
+And here's how to do it with the CLI:
+
+First, configure the
+[DVC Studio Access Token](https://dvc.org/doc/studio/user-guide/account-and-billing#studio-access-token)
+(this only needs to be done once):
+
+```console
+dvc config --global studio.token <your Studio token>
```

Comment:
Let's remember to update this once https://github.com/iterative/dvc.org/pull/4981 is merged.

Reply:
Captured in #4990